front/components/data_source/IntercomConfigView.tsx

```diff
@@ -0,0 +1,86 @@
+import { ContextItem, IntercomLogo, SliderToggle } from "@dust-tt/sparkle";
+import type { APIError, DataSourceType, WorkspaceType } from "@dust-tt/types";
+import { useContext, useState } from "react";
+import * as React from "react";
+
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { useConnectorConfig } from "@app/lib/swr/connectors";
+
+export function IntercomConfigView({
```

Comment:
This component was extracted from front/pages/w/[wId]/builder/data-sources/[name]/index.tsx, which is great!

Why keeping the component duplicated? We could have front/pages/w/[wId]/builder/data-sources/[name]/index.tsx calling this component from here. This would even help a lot with the diff of the PR to see what code is new and what was moved!

Reply:
You're absolutely right, I thought I actually did it 🙈

Reply:
You right, and actually the whole `front/pages/w/[wId]/builder/data-sources/[name]/index.tsx` needs to go to trash!

Reply:
Nice!!